Guide/DynamicProgramming/Abbreviation/README.md

```diff
@@ -0,0 +1,115 @@
+# Abbreviation
+
+## Problem
+This problem comes from [HackerRank](https://www.hackerrank.com/contests/world-codesprint-6/challenges/abbr).
+Category: Dynamic Programming
+Difficulty: Medium
+
+### Overview
+Read the link instructions,
+they are helpful when solving
+greater than eye deep
+
+Edge cases and complexities are covered below. Please read the problem first.
```

Comment:
Not really, but I wasn't sure if I should just delete the whole section since it's somewhat irrelevant.